src/app/legacy/psammead/psammead-story-promo/src/__snapshots__/index.test.jsx.snap

```diff
@@ -2454,24 +2454,24 @@ exports[`StoryPromo should render a RTL Live promo correctly 1`] = `
   color: #222222;
   margin: 0;
   padding-bottom: 0.5rem;
-  font-family: ReithSerif,Helvetica,Arial,sans-serif;
+  font-family: "BBC Reith Qalam","Times New Roman",Arial,Verdana,Geneva,Helvetica,sans-serif;
```

Comment:
The test was incorrect - it was not using an RTL service before, but now it is (arabic), so the font info & line heights are larger.